### PR TITLE
Fullwidth no title

### PR DESCRIPTION
**Pull request by adiraomj**
Implemented enhancement #28

### CODE REVIEW

File: page-full-no-title.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * The template for displaying full width pages without the title section.
+ *
+ * @package focus
+ * @since focus 1.0
```
Comment (Misplon) [inline page-full-no-title.php]: @adiraomj Should we change this to `1.3.3`?

**Comment by adiraomj** [comment]
Added meta box with checkbox option to use the first panels row as the page header.

@Misplon please have a look.

**Comment by Misplon** [comment]
Looking awesome!! We'll probably need to add a note about this being page template dependent. How about:

`Moves first row into the header. Page template must be set to Full Width - No Title.`

**Comment by Misplon** [comment]
I'll make sure the documentation is up to spec for Focus once we get this out. I also want to document how you can automate video thumbnails using one of the video thumbnail plugins. I once did a casual tutorial on the forum but it will be cool to have it in the docs.

**Comment by adiraomj** [comment]
Done. Please wait till @gregpriday check's the code before merging thanks. Just to make sure that I have not broken anything. Thanks ;)

**Comment by Misplon** [comment]
@gregpriday We've decided to show the meta box always and not conditionally based on the Full Width - No Page Title template being selected. The thinking being to let users know the option is there. You good with that?

**Comment by gregpriday** [comment]
@Misplon Yeah, that's cool. Just a note should be good enough.

**Pull request merged**